### ISSUE TITLE
Unable to create deployment connection to Sandbox

### ISSUE THREAD

**Issue by klouf-mp** [joeferraro/MavensMate#514]
OS X 10.10.4
MavensMate v0.0.1-rc02

When trying to add a deployment connection to a sandbox, am consistently getting "INVALID_LOGIN: Invalid username, password, security token; or user locked out." with different sandboxes...even after resetting pws/security tokens.

Was scratching my head over this one, and then found that I could toggle the core developer tools, which let me see the request info when attempting to add the connection (actually was able to see this in the log as well).

After looking at the request/payload, when creating a deployment connection, the "Org Type" field/parameter name in the payload is specified/sent as "org_type". After looking through the source files, it looked to me that everything was actually looking for "orgType". So I fired up Postman to test my theory of incorrect parameter name. First set the URL and payload the same -- as expected, same invalid login result. Changed the "Org Type" parameter name to "orgType" and voila, success!

Apologies if I'm wrong, but it appears the issue lies on line 627 (and also on 456) of /lib/mavensmate/ui/templates/deploy/index.html ([here](https://github.com/joeferraro/MavensMate/blob/master/lib/mavensmate/ui/templates/deploy/index.html#L627) and [here](https://github.com/joeferraro/MavensMate/blob/master/lib/mavensmate/ui/templates/deploy/index.html#L456)) and also similarly appears in the Project Upgrade page -- line 74 of /lib/mavensmate/ui/templates/project/upgrade.html ([here](https://github.com/joeferraro/MavensMate/blob/master/lib/mavensmate/ui/templates/project/upgrade.html#L74))

tl;dr "orgType" seems to be incorrectly passed as "org_type" in some places.

**Comment by klouf-mp** [comment]
Ack, went to deploy and got same error, looked some more and found a third place with the same issue:
Line 456 in /lib/mavensmate/ui/templates/deploy/index.html ([here](https://github.com/joeferraro/MavensMate/blob/master/lib/mavensmate/ui/templates/deploy/index.html#L456))
